### DISCUSSION TITLE
Turning OFF the injector

### DISCUSSION THREAD

**Discussion by Rusinio** [winstxnhdw/lc-hax#575] [Ideas]
Closing the game it's the only solution i've found for closing the injector completely, it's well received that you can open it at any time in the game but not close it. Would be very useful to have a feature or just a way to disable the injector while in game.

**Comment by winstxnhdw** [comment]
Might be possible but I am not sure how it'll look like. I'll have to look at other injectors for ideas. Either way, I don't think I have the time to invest into this right now, especially given it's low priority.

**Reply by Rusinio** [reply]
Understandable, thanks for taking it in mind!
